testsuite/single_node_performance.py

```diff
@@ -90,6 +91,10 @@ class RunGroupConfig:
 # https://app.axiom.co/aptoslabs-hghf/explorer?qid=29zYzeVi7FX-s4ukl5&relative=1
 # fmt: off
 TESTS = [
+    RunGroupConfig(expected_tps=10000, key=RunGroupKey("econia-basic1-market"), expected_stages=10, included_in=Flow.ECONIA),
+    RunGroupConfig(expected_tps=10000, key=RunGroupKey("econia-advanced1-market"), expected_stages=10, included_in=Flow.ECONIA),
```

Comment:
Why is expected_stages = 10 when we have on 4 stages?

Reply:
so it's more than enough. I'll make that boolean, and search until I find a missing stage